.github/workflows/pr-ci.yaml

```diff
@@ -0,0 +1,37 @@
+
+# This workflow will install Python dependencies, run tests and lint with a multiple versions of Python
+name: CloudFormation CLI CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    env:
+      AWS_DEFAULT_REGION: us-east-1
+    runs-on: ubuntu-latest
```

Comment:
we should test on `macos-latest` and `windows-latest` as well

Reply:
agree although was just shooting for parity with the current job in this PR. can see how easy would be to use  in addition. are we trying to do 9 jobs in total? os X python versions?

Reply:
Windows [fails hard](https://github.com/aws-cloudformation/cloudformation-cli/pull/654/checks?check_run_id=1567545887). 

Are you OK with creating issues to follow up for windows? I'll try without

Reply:
Got it fixed. just had to remove attempting to upgrade pip